ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyMzYzIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJpdF9JVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2NvdXBlL2NsZS9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyJ9LCJmbGFncyI6eyJwYWdlVmlldyI6dHJ1ZX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTE0NjQyOTMwNC4xNzQ1NDg4NzgyIn0sImFub25JZCI6Ijg2ZGJjMmM0ZWIwNjM3NGQifSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoxfSwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfSwiY2F0YWxvZyI6eyJQcm9kdWN0Ijp7Il9pZCI6IjIzNjMiLCJkaW1lbnNpb25zIjp7IkJhdW11c3RlclNob3J0IjpbIjIzNjMiXX19fSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI0NjUxMjkwNzI1OTIwNzMwMyJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x558a34a78cea <unknown>
#1 0x558a345295f0 <unknown>
#2 0x558a3450fe86 <unknown>
#3 0x558a3450e79a <unknown>
#4 0x558a3450eeff <unknown>
#5 0x558a3450ee54 <unknown>
#6 0x558a3452d7ff <unknown>
#7 0x558a345d29aa <unknown>
#8 0x558a345a0662 <unknown>
#9 0x558a345c6660 <unknown>
#10 0x558a345a0433 <unknown>
#11 0x558a3456cea3 <unknown>
#12 0x558a3456db01 <unknown>
#13 0x558a34a3db3b <unknown>
#14 0x558a34a41a21 <unknown>
#15 0x558a34a24c32 <unknown>
#16 0x558a34a42594 <unknown>
#17 0x558a34a08eef <unknown>
#18 0x558a34a66d98 <unknown>
#19 0x558a34a66f76 <unknown>
#20 0x558a34a77b36 <unknown>
#21 0x7f9557e9caa4 <unknown>
#22 0x7f9557f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTE0NjQyOTMwNC4xNzQ1NDg4NzgyIn0sImFub25JZCI6Ijg2ZGJjMmM0ZWIwNjM3NGQifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvY291cGUvY2xlL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMzQ0OTMzMzM5MDY2MTU0NCIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x558a34a78cea <unknown>
#1 0x558a345295f0 <unknown>
#2 0x558a3450fe86 <unknown>
#3 0x558a3450e79a <unknown>
#4 0x558a3450eeff <unknown>
#5 0x558a3450ee54 <unknown>
#6 0x558a3452d7ff <unknown>
#7 0x558a345d29aa <unknown>
#8 0x558a345a0662 <unknown>
#9 0x558a345c6660 <unknown>
#10 0x558a345a0433 <unknown>
#11 0x558a3456cea3 <unknown>
#12 0x558a3456db01 <unknown>
#13 0x558a34a3db3b <unknown>
#14 0x558a34a41a21 <unknown>
#15 0x558a34a24c32 <unknown>
#16 0x558a34a42594 <unknown>
#17 0x558a34a08eef <unknown>
#18 0x558a34a66d98 <unknown>
#19 0x558a34a66f76 <unknown>
#20 0x558a34a77b36 <unknown>
#21 0x7f9557e9caa4 <unknown>
#22 0x7f9557f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjExNDY0MjkzMDQuMTc0NTQ4ODc4MiJ9LCJhbm9uSWQiOiI4NmRiYzJjNGViMDYzNzRkIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2NvdXBlL2NsZS9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJwZXJmb3JtYW5jZSI6eyJzZGtQYXJzZVRpbWUiOjF9LCJfdG9vbHNFdmVudExpbmtJZCI6IjM0Nzk2MjkxMTM3NzUyNTkiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x558a34a78cea <unknown>
#1 0x558a345295f0 <unknown>
#2 0x558a3450fe86 <unknown>
#3 0x558a3450e79a <unknown>
#4 0x558a3450eeff <unknown>
#5 0x558a3450ee54 <unknown>
#6 0x558a3452d7ff <unknown>
#7 0x558a345d29aa <unknown>
#8 0x558a345a0662 <unknown>
#9 0x558a345c6660 <unknown>
#10 0x558a345a0433 <unknown>
#11 0x558a3456cea3 <unknown>
#12 0x558a3456db01 <unknown>
#13 0x558a34a3db3b <unknown>
#14 0x558a34a41a21 <unknown>
#15 0x558a34a24c32 <unknown>
#16 0x558a34a42594 <unknown>
#17 0x558a34a08eef <unknown>
#18 0x558a34a66d98 <unknown>
#19 0x558a34a66f76 <unknown>
#20 0x558a34a77b36 <unknown>
#21 0x7f9557e9caa4 <unknown>
#22 0x7f9557f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2NvdXBlL2NsZS9vdmVydmlldy5odG1sIiwidmVoaWNsZUNvZGUiOiJjMjM2IiwidmVoaWNsZUJNIjoiMjM2MyIsInZlaGljbGVQSUhpc3RvcnkiOiIyMzYzIiwidGltZXN0YW1wIjoxNzQ1NDg4Nzg3OTMyLCJleHBpcmF0aW9uIjoxMjB9fX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTE0NjQyOTMwNC4xNzQ1NDg4NzgyIn0sImFub25JZCI6Ijg2ZGJjMmM0ZWIwNjM3NGQifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvY291cGUvY2xlL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI2NTQ4MTI0NDQ2MzcwMjQ5IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x558a34a78cea <unknown>
#1 0x558a345295f0 <unknown>
#2 0x558a3450fe86 <unknown>
#3 0x558a3450e79a <unknown>
#4 0x558a3450eeff <unknown>
#5 0x558a3450ee54 <unknown>
#6 0x558a3452d7ff <unknown>
#7 0x558a345d29aa <unknown>
#8 0x558a345a0662 <unknown>
#9 0x558a345c6660 <unknown>
#10 0x558a345a0433 <unknown>
#11 0x558a3456cea3 <unknown>
#12 0x558a3456db01 <unknown>
#13 0x558a34a3db3b <unknown>
#14 0x558a34a41a21 <unknown>
#15 0x558a34a24c32 <unknown>
#16 0x558a34a42594 <unknown>
#17 0x558a34a08eef <unknown>
#18 0x558a34a66d98 <unknown>
#19 0x558a34a66f76 <unknown>
#20 0x558a34a77b36 <unknown>
#21 0x7f9557e9caa4 <unknown>
#22 0x7f9557f29c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-App-Allure/QA-App-Allure/Tests/BFV1_attempt_1.png'